app/controllers/groups_controller.rb

```diff
@@ -468,12 +468,11 @@ def group_params(automatic: false)
     params.require(:group).permit(*permitted_params)
   end
 
-  def find_group(param_name)
+  def find_group(param_name, ensure_can_see = true)
```

Comment:
I prefer to use keyword arguments now a days because `find_group(:name, ensure_can_see: true)` is clearer than `find_group(:name, true)`

Reply:
Done.